### PR TITLE
[WIP] Update gems because of potential security vulnerabilities

### PR DESCRIPTION
**Pull request by macejmic**
**What this PR does / why we need it**:

We found potential security vulnerabilities in our dependencies.

### CODE REVIEW

**Comment by gsaslis** [comment]
please note #44 is related to this and still pending (and it is also a blocking task for completing the open sourcing - and therefore any subsequent work we want to do on the open source repo, such as  work on upstream builds, etc. )

File: Gemfile.base [hunk 1]
```diff
@@ -167,8 +167,8 @@ group :development do
 
   # Rails 4.2 have it built in
   gem 'quiet_assets'
-
-  gem 'yard', require: false
+ 
+  gem 'yard', '~> 0.9.11', require: false
```
Comment (mikz) [inline Gemfile.base]: Why do we need to pin to a patch version? `~> 0.9` should be enough no? Then `bundle update` would keep working as expected.

**Comment by Martouta** [comment]
This PR is probably not needed anymore now that we have [dependabot](https://github.com/3scale/porta/pulls/app%2Fdependabot).
WDYT? @macejmic @hallelujah @guicassolato @3scale/system

**Comment by gsaslis** [comment]
@Martouta https://github.com/3scale/porta/network/alerts says otherwise

**Comment by Martouta** [comment]
@gsaslis @macejmic Ok then perhaps rebasing and merging, or closing it and leaving this for once we have Rails 5 merged and the Gemfile.prod extracted.
My main poin is simply to have the less abandoned PRs as possible 😄 
And if you guys consider it necessary, let's also open a Jira issue for it 🤷‍♀️

**Comment by Martouta** [comment]
Now really not needed for vulnerabilities anymore 😄

**Pull request closed without merging**